types/globals/index.d.ts

```diff
@@ -7,9 +13,11 @@ interface ConfigType {
 }
 
 declare global {
-    namespace NodeJS {
-      interface Global {
-        ALLOWEDLIST: strign[];
-      }
+  namespace NodeJS {
+    interface Global {
+      ALLOWEDLIST: strign[];
```

Comment:
strign?

Reply:
Interesting, that ts didn't tell anything about it. Nice catch!